generator/src/main/java/com/scottlogic/deg/generator/validators/GenerationConfigValidator.java

```diff
@@ -19,6 +32,42 @@ public ValidationResult validateCommandLine(GenerationConfig config) {
             errorMessages.add("RANDOM mode requires max row limit: use -n=<row limit> option");
         }
 
+        if (!configSource.overwriteOutputFiles()) {
+            if (configSource.shouldViolate()) {
+                checkViolationGenerateOutputTarget(errorMessages, outputTarget);
+            } else {
+                checkGenerateOutputTarget(errorMessages, outputTarget);
+            }
+        }
+
         return validationResult;
     }
+
+    private void checkGenerateOutputTarget(ArrayList<String> errorMessages, OutputTarget outputTarget) {
+        if (outputTarget.isDirectory()) {
+            errorMessages.add("Invalid Output - target is a directory, please use a different output filename or use the --overwrite option");
+        } else if (outputTarget.exists()) {
+            errorMessages.add("Invalid Output - file already exists, please use a different output filename or use the --overwrite option");
+        }
+    }
+
+    private void checkViolationGenerateOutputTarget(ArrayList<String> errorMessages, OutputTarget outputTarget) {
+        if (!outputTarget.isDirectory()) {
+            errorMessages.add("Invalid Output - not a directory. please enter a valid directory name or use the --overwrite option");
```

Comment:
`--overwrite` isn't applicable here, it must be a directory

Reply:
We should also check if any of the violation output files exist?
i.e. `001.csv` or `01.json`

Reply:
are those the only files we need to check for?
should we make sure the directory is empty?

Reply:
or should we check for the existence of any .csv and json files with numbers as their name part?